package.json

```diff
@@ -40,8 +40,8 @@
     "typescript": "^4.7.3"
   },
   "dependencies": {
-    "kafkajs": "^2.0.2",
-    "kafkajs-zstd-node-16": "^0.1.1",
+    "@kafkajs/zstd": "^0.1.1",
+    "kafkajs": "^2.2.4",
```

Comment:
Why do we need to update the minimum version of kafkajs?

Reply:
Oops, we dont, I ran an upgrade command but this still works with 2.0.2. Reverted in latest commit